apps/site/src/pages/api/types/entity-type/shared/schema.ts

```diff
@@ -1,8 +1,9 @@
-import {
-  ENTITY_TYPE_META_SCHEMA,
-  EntityType,
-  EntityTypeWithMetadata,
-} from "@blockprotocol/graph";
+/*
+ @todo - We should be able to import from the `@blockprotocol/graph` package here but we're running into strange errors
+   with named exports, CommonJS, and ESM modules from the dependency on `/core`
+ */
+import { EntityType, EntityTypeWithMetadata } from "@blockprotocol/graph";
+import { ENTITY_TYPE_META_SCHEMA } from "@blockprotocol/type-system/slim";
```

Comment:
Doing this fixed the `seed-db` script which was causing problems with `ts-node` not understanding the `@blockprotocol/core` package